drivers/sshj/pom.xml

```diff
@@ -87,15 +87,9 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>net.schmizz</groupId>
+      <groupId>com.hierynomus</groupId>
       <artifactId>sshj</artifactId>
-      <version>0.8.1</version>
-    </dependency>
-    <!-- required by sshj -->
-    <dependency>
```

Comment:
No longer needed as a separate dep, as sshj now [includes it](https://github.com/hierynomus/sshj/blob/v0.12.0/pom.xml#L96-L99)